07-Extract-NDVI-From-Rasters-in-R.Rmd

```diff
@@ -35,43 +36,45 @@ rasters in `R`.
+* Save summary values to a .csv file.
+* Plot summary pixel values using `ggplot()`. 
 
-* How to extract summary pixel values from a raster.
-* 
 
 ###Things You'll Need To Complete This Lesson
 
+Please be sure you have the most current version of `R` and, preferably,
+RStudio to write your code.
+
 ###R Libraries to Install:
 
 * **raster:** `install.packages("raster")`
 * **rgdal:** `install.packages("rgdal")`
 
-
-####Tools To Install
-
-Please be sure you have the most current version of `R` and preferably
-R studio to write your code.
-
-
 ####Data to Download
+Download the NDVI raster files in this teaching dataset:
 
-Download the raster files for the Harvard Forest dataset:
-
-<a href="http://files.figshare.com/2434040/NEON_RemoteSensing.zip" class="btn btn-success"> DOWNLOAD Sample NEON Airborne Observation Platform Raster Data</a> 
+<a href=" https://ndownloader.figshare.com/files/3579867" class="btn btn-success"> Download Landsat derived NDVI raster files</a> 
 
-The LiDAR and imagery data used to create the rasters in this dataset were 
-collected over the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank" >Harvard</a> and 
-<a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >San Joaquin</a> field sites 
-and processed at <a href="http://www.neoninc.org" target="_blank" >NEON </a> 
-headquarters. The entire dataset can be accessed by request from the 
-<a href="http://www.neoninc.org/data-resources/get-data/airborne-data" target="_blank"> NEON 
-website.</a>  
+The imagery data used to create the rasters in this dataset were collected over
+the National Ecological Observatory Network's
+<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank" >Harvard</a>
+and 
+<a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank" >San Joaquin</a>
+field sites.  
+The imagery was created by the U.S. Geological Survey (USGS) using a 
+<a href="http://eros.usgs.gov/#/Find_Data/Products_and_Data_Available/MSS" target="_blank" >  multispectral scanner</a>
+on a <a href="http://landsat.usgs.gov" target="_blank" > Landsat Satellite </a>.
+The data files are Geographic Tagged Image-File Format (GeoTIFF). 
 
-####Recommended Pre-Lesson Reading
 
-* <a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
-Read more about the `raster` package in R.</a>
+####Setting the Working Directory
+The code in this lesson assumes that you have set your working directory to the
+location of the unzipped file of data downloaded above.  If you would like a
+refresher on setting the working directory, please view the [Setting A Working Directory In R]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson") 
+lesson prior to beginning this lesson.
 
 ####Raster Lesson Series 
 This lesson is a part of a series of raster data in R lessons:
```

Comment:
To make this section shorter on each page, i think we can just have a one sentence link to the workshop page.